.github/workflows/build-and-deploy.yml

```diff
@@ -205,6 +205,33 @@ jobs:
         with:
           password: ${{ secrets.PYPI_TOKEN }}
 
+  deploy-rust:
+    runs-on: kuzu-self-hosted-testing
+    env:
+      CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Update Cargo.toml version
+        run: python3 update_version.py
+        working-directory: tools/rust_api
+
+      - name: Deploy crate to Crates.io
+        run: cargo publish --allow-dirty
+        if: ${{ github.event.inputs.isDeploy == 'true' }}
+        working-directory: tools/rust_api
+
+      - name: Test publishing crate
+        run: cargo publish --dry-run --allow-dirty
+        if: ${{ github.event.inputs.isDeploy != 'true' }}
+        working-directory: tools/rust_api
+
+      - name: Upload crate
+        uses: actions/upload-artifact@v3
+        with:
+          name: kuzu-deploy-crate
+          path: target/package/*.crate
```

Comment:
It seems that the path here is not correct. I tested run this job once and it’s not able to find any artifact. https://github.com/kuzudb/kuzu/actions/runs/5522371969/jobs/10071856122

Reply:
Whoops. I wrote it relative to `tools/rust_api`, but that's not the working directory for that step.